tracing-subscriber/src/filter/mod.rs

```diff
@@ -6,16 +6,20 @@
 //!
 //! [`layer`  module documentation]: crate::layer#filtering-with-layers
 //! [`Layer`]: crate::layer
+mod directive;
 #[cfg(feature = "env-filter")]
 mod env;
 mod filter_fn;
 #[cfg(feature = "registry")]
 mod layer_filters;
 mod level;
+mod targets;
 
+pub use self::directive::ParseError as DirectiveParseError;
```

Comment:
If you need to do this rename even internally, maybe might as well just call the thing `DirectiveParseError`?